evmlab/vm.py

```diff
@@ -142,12 +144,28 @@ class CppVM(VM):
     def canonicalized(output):
         from . import opcodes
         valid_opcodes = opcodes.reverse_opcodes.keys()
-        steps = []
+        logger.debug(output)
 
+        steps = []
         for x in output:
-            if x[0:2] == "[{":
-                steps = json.loads(x)
-            logger.debug(output)
+            try:
+                if x[0:2] == "[{":
+                        steps = json.loads(x)
+                        steps = []
```

Comment:
How can this possibly work - you're clearing `steps` here ?

Reply:
oops, its not working. Didn't notice yet because I'm trying to get cpp to print a stateroot